Code/GraphMol/FileParsers/XYZFileWriter.cpp

```diff
@@ -28,6 +28,9 @@ std::string MolToXYZBlock(const ROMol& mol, int confId) {
   std::stringstream ss;
   ss << nAtoms << '\n';
 
+  std::stringstream formatString;
+  formatString << "%-3s %11." << precision << "f %11." << precision << "f %11." << precision << "f\n";
```

Comment:
I'm not convinced that the `11` should always be there.
It's not wrong (since it's the minimum field width), but it does end up looking a bit odd when precision is small:
```
In [9]: print(Chem.MolToXYZBlock(ms[0],precision=3))
18
23684363
Na        2.000       2.767       0.000
O         8.220      -1.689       0.000
O         8.756      -0.042       0.000
O         2.588       1.958       0.000
O         3.989       2.976       0.000
N         6.252      -0.990       0.000
C         5.752       0.549       0.000
C         6.752       0.549       0.000
C         5.165       1.358       0.000
C         5.443      -0.402       0.000
C         7.061      -0.402       0.000
C         4.170       1.254       0.000
C         7.340       1.358       0.000
C         4.492      -0.711       0.000
C         8.012      -0.711       0.000
C         3.582       2.063       0.000
C         9.171      -1.998       0.000
C         9.379      -2.976       0.000
```
To be consistent with what's already there, this could be theoretically be `precision+5`